Gems/Atom/Feature/Common/Assets/Materials/Pipelines/LowEndPipeline/TintedTransparent_StandardLighting.shader.template

```diff
@@ -0,0 +1,52 @@
+{
+    "Source" : "INSERT_AZSL_PATH_HERE",
+
+    "DepthStencilState" :
+    {
+        "Depth" :
+        {
+            "Enable" : true,
+            "CompareFunc" : "GreaterEqual",
+            "writeMask": "Zero"
```

Comment:
Note to reviewers: writeMask is different between the **[tinted]transparent** and **opaque** .shader.template files